win32_event_log/README.md

```diff
@@ -252,7 +272,39 @@ The legacy method is the default mode for filters.
 <!-- xxz tab xxx -->
 <!-- xxx tab "Logs" xxx -->
 
-For each filter, add a log processing rule in the configuration file at `win32_event_log.d/conf.yaml`.
+You can use the `query`, as well as the `log_processing_rules` regex option, to filter event logs. If you are using the Logs Tailer, the `query` option has better filtering performance than the `log_processing_rules` filters. The Datadog Agent cannot see any events filtered out by the `query` option. When using the `log_processing_rules` filters, the Agent can see every event and must process and format the event before applying the regex filter, so it takes more processing, and is less performant.
```

Comment:
> The Datadog Agent cannot see any events filtered out by the `query` option.

I'm wondering if this might be confusing.

> If you are using the Logs Tailer,

This is in the logs tab, do we need to specify this?

maybe something like

```suggestion
You can use the `query`, as well as the `log_processing_rules` regex option, to filter event logs. The `query` option has better filtering performance than the `log_processing_rules` filters. When using the `log_processing_rules` filters, the Agent must process and format each event before applying the regex filter, so it takes more processing, and is less performant.
```